.github/workflows/ci.yml

```diff
@@ -8,3 +8,161 @@ on:
 jobs:
   ci:
     uses: killbill/gh-actions-shared/.github/workflows/ci.yml@main
+  e2e:
+    runs-on: ubuntu-latest
+    services:
+      mysql:
+        image: mysql:8.0
+        env:
+          MYSQL_ROOT_PASSWORD: root
+        ports:
+        - 3306:3306
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=10
+    strategy:
+      matrix:
+        include:
+          - ruby-version: '2.4.2'
+            ref: refs/heads/master
+            database-adapter: 'mysql2'
+            database-user: 'root'
+            database-password: 'root'
+            database-port: '3306'
+            database: 'mysql'
```

Comment:
Seems like the job supports different values for `database` but all properties here are targeted for `mysql` - Am i missing something?

Reply:
PostgreSQL support is for future work 😄 

We have it enabled on the other repo (https://github.com/killbill/killbill-integration-tests/actions/runs/1863918667), but I just wanted to see how stable the job is here, as the setup is a tad different.